conf/evolutions/default/85.sql

```diff
@@ -1,6 +1,12 @@
 # --- !Ups
 ALTER TABLE `users`
-ADD COLUMN `passwordCreatedDate` DATETIME NOT NULL AFTER `password`;
+
+ADD COLUMN `passwordCreatedDate` DATETIME NOT NULL AFTER `password`,
+ADD COLUMN `date_created` DATETIME NOT NULL AFTER `last_login`,
+ADD COLUMN `created_by` INT NOT NULL AFTER `creation_date`;
```

Comment:
`ADD COLUMN `created_by` INT NOT NULL AFTER `creation_date`;`

Should probably be:

`ADD COLUMN `created_by` INT NOT NULL AFTER `date_created`;`